Dockerfile

```diff
@@ -26,3 +26,7 @@ COPY Gemfile Gemfile.lock $INSTALL_PATH
 RUN bundle install
 
 COPY . $INSTALL_PATH
+
+RUN mkdir -p /etc/nginx/ssl
+
+COPY nginx/ssl/ /etc/nginx/ssl/
```

Comment:
nginx/ssl/ required

For security reasons, docker can not break out of build working directory.